framework/platform/options.cpp

```diff
@@ -24,24 +24,33 @@
 
 namespace vkb
 {
-Options::Options(const std::string &usage, const std::vector<std::string> &args) :
-    usage{usage},
-    parse_result{docopt::docopt(usage, args, false)}
+void Options::parse(const std::string &usage, const std::vector<std::string> &args)
+
 {
+	if (usage.size() != 0)
+	{
+		this->usage        = usage;
+		this->parse_result = docopt::docopt(usage, args, false);
+	}
+	else
+	{
+		LOGW("Application usage empty");
```

Comment:
Would this warning be printed every time we run one of the sample entry points?

Reply:
Yes

Reply:
I think we should remove the warning them, it's a normal use case to use the sample entry points. An empty usage can never happen when calling the main `vulkan_best_practice` app, if I'm not mistaken.

Reply:
I had a warning for the meantime in the hotfix, as I was planning on changing the entire entry-point template for samples to be separate to Vulkan Best Practices so they don't even parse the arguments (i.e. you shouldn't be calling parse arguments if there is no usage, hence the warning).

I guess this also works though, I will remove it.